### PR TITLE
“the worker's Documents” ➡ “owner set”

### PR DESCRIPTION
**Pull request by sideshowbarker**
This aligns with the changes made in https://github.com/whatwg/html/pull/2520
https://github.com/whatwg/html/commit/59a4750f475acd789ee436b4906972ba2081d8b3

### CODE REVIEW

**Comment by annevk** [comment]
This change correctly rewrite what was there before, which is probably a good first step. But I'm not sure what was there actually makes sense.

**Comment by sideshowbarker** [comment]
>  But I'm not sure what was there actually makes sense.

Me neither. So I guess this an chance to get some fresh review of the requirements in this paragraph

Of course feel free to add more requested reviewers. On my own I’m not sure who to pick

**Comment by jungkees** [comment]
It should have been my bad having misunderstood the worker's Document back then. I'll find time to figure out what we should do to fix it.

**Comment by domenic** [comment]
Oh, I didn't see this. In #1130 I just updated it to be a vague "the worker's owners". I guess I should not do that given that we're unsure what's going on.

**Comment by annevk** [comment]
I think that's fine as at least it's not a broken link. We can then file a new issue on defining the lifetime of service workers and what changes to HTML might be needed for that.

**Comment by annevk** [comment]
(And what changes to Service workers...)

**Comment by ylafon** [comment]
Marked as non-substantive for IPR from ash-nazg.

File: docs/index.bs [hunk 1]
```diff
@@ -243,7 +243,7 @@ spec: webappsec-referrer-policy; urlPrefix: https://w3c.github.io/webappsec-refe
...
 
-    When the request's [=request/url=] <a>is local</a>, if the [=/service worker client=]'s <a>responsible browsing context</a> is a <a>nested browsing context</a> or the [=/service worker client=] is a <a>worker client</a>, the [=/service worker client=] inherits the [=/service worker registration=] from its <a>parent browsing context</a>'s environment or one of <a>the worker's Documents</a>' environment, respectively, if it exists.
+    When the request's [=request/url=] <a>is local</a>, if the [=/service worker client=]'s <a>responsible browsing context</a> is a <a>nested browsing context</a> or the [=/service worker client=] is a <a>worker client</a>, the [=/service worker client=] inherits the [=/service worker registration=] from its <a>parent browsing context</a>'s environment or from the environment of a {{Document}} in the [=/service worker client=]'s <a>active service worker</a>'s [=owner set=], respectively, if it exists.
```
Comment (jungkees) [inline docs/index.bs]: I'll sort out the SW inheritance anyway, but this line should be:
s/\<a>active service worker\</a>/[=environment settings object/global object=]/.

**Comment by jungkees** [comment]
@sideshowbarker, could you address https://github.com/w3c/ServiceWorker/pull/1124#pullrequestreview-35046470 and rebase?

**Comment by sideshowbarker** [comment]
> @sideshowbarker, could you address #1124 (review) and rebase?

Yup, done

**Comment by jungkees** [comment]
Thanks. I'll merge this and put some additional change: relevant owner concept from HTML can be used to simply it I think.

> I think that's fine as at least it's not a broken link. We can then file a new issue on defining the lifetime of service workers and what changes to HTML might be needed for that.

https://github.com/w3c/ServiceWorker/issues/765 and https://github.com/whatwg/html/pull/2080 track the relevant spec issues and efforts.

**Pull request merged**